source/manual/data-gov-uk-operations.html.md

```diff
@@ -24,18 +24,27 @@ This will populate a new index and rotate the `dataset-staging` alias to point t
 
 ## Sync from [CKAN]
 
-This is done automatically using Sidekiq Scheduler. You can run manually to populate your local DB.
+### Perform a full re-sync
+
+The sync is normally done automatically using Sidekiq Scheduler. There may be times when you need to throw away the existing Postgres database, sync all datasets from CKAN and reindex.
+
+This will not make any changes to the content on Find until the reindex has completed and the Elastic index is updated.  This will affect data served on Publish, however this service is not currently used for publishing or editing datasets. 
 
 ```
-## make sure your local DB is empty
-rails db:drop db:setup
+### connect to staging
+cf ssh publish-data-beta-staging
+### connect to production
+cf ssh publish-data-beta-production
+
+## make sure the database is empty
+/tmp/lifecycle/launcher /home/vcap/app 'rails db:drop db:setup' ''
 
 ## make sure your local index is setup
```

Comment:
These docs are now about production/staging, so it's not "your local index"

Reply:
Good point.  I've updated this now.